R/nested.R

```diff
@@ -37,6 +36,18 @@ compute_parse_data_nested <- function(text) {
   pd_nested
 }
 
+#' Obtain token from text
+#'
+#' [utils::getParseData()] is used to obtain a flat parse table from `text`.
+#' @param text A character vector.
+#' @return A flat parse table
+tokenize <- function(text) {
+  parsed <- parse(text = text, keep.source = TRUE)
+  parse_data <- tbl_df(utils::getParseData(parsed, includeText = TRUE))
```

Comment:
I think `as_tibble()` is preferred these days. Do we still need `includeText = TRUE`?